snapshot-labs/sx-monorepo
# fix: truncate space breadcrumb

### Summary



This PR truncates the space breadcrumb if too long

Before fix

![Screenshot 2024-09-07 at 18 32 34](https://github.com/user-attachments/assets/3f041272-b51a-4f57-98a3-1a12cd3e4802)

After fix

![Screenshot 2024-09-07 at 18 32 15](https://github.com/user-attachments/assets/118a579b-2633-4072-8dbc-8cc1196b06d6)


### How to test

1. Go to http://localhost:8080/#/sep:0xb58B05A1c8263441Bd74454544fD66CE25D300AD/proposal/58
2. It should truncate the long space title, and show the account and theme switch button on the top right
